src/App.css

```diff
@@ -0,0 +1,20 @@
+.snackbar {
+  @apply invisible border-r-8 bg-slate-700 text-white text-center p-2 fixed z-10 bottom-[30px] min-w-[250px] ml-[-125px] left-[50%]
```

Comment:
remove this aliasing as well.

Reply:
As I said in Slack thread, its causing some issues.

Reply:
Then I recommend fixing the issue or removing animation altogether if it wasn't mentioned in the assignment.

Reply:
removed animation. Fixed.